vlm/src/main/scala/geotrellis/contrib/vlm/cache/LazyCache.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.contrib.vlm.cache
+
+import com.github.blemale.scaffeine.Cache
+import scala.collection.concurrent.TrieMap
+
+/** A limited cache wrapper that can be "disable" caching (lazy instance creation) */
+case class LazyCache[K, V](getUnderlying: () => Cache[K, V], enabled: Boolean = true) {
```

Comment:
Is it lazy or just configuration wrapper? Name is just weird.

Reply:
It's lazy, there would be no cache instance created if that would be create with a param `enabled = false`. Hm, do you think smth like `DisableableCache` sounds better? O: